deploy/ods-pipeline/charts/ods-pipeline-images/templates/bc-ods-buildah.yaml

```diff
@@ -11,22 +11,10 @@ spec:
   resources: {}
   successfulBuildsHistoryLimit: 5
   failedBuildsHistoryLimit: 5
+  postCommit: {}
   strategy:
     type: Docker
-    dockerStrategy:
-      dockerfilePath: build/package/Dockerfile.buildah
-      from:
-        kind: DockerImage
-        name: 'registry.redhat.io/ubi8:8.4'
-      buildArgs:
-        - name: aquasecScannerUrl
-          value: '{{.Values.aquasecScannerUrl}}'
-      pullSecret:
-        name: registry.redhat.io
-  postCommit: {}
   source:
-    type: Git
-    git:
-      uri: '{{.Values.odsPipelineGitRepoUri}}'
-      ref: '{{.Values.odsPipelineGitRepoRef}}'
+    dockerfile: |-
+      FROM ghcr.io/opendevstack/ods-pipeline/ods-buildah:{{.Values.global.imageTag}}
```

Comment:
Could we render a file here? See e.g. https://helm.sh/docs/chart_template_guide/accessing_files/#helm.

That would allow us to use the Dockerfile also locally for those that need to wrap it. 

We can also do that change in a second step though ...